packages/@sanity/desk-tool/src/pane/DocumentsPane.js

```diff
@@ -115,16 +140,27 @@ export default withRouterHOC(class DocumentsPane extends React.PureComponent {
     })
   }
 
-  handleGoToCreateNew = () => {
-    const {selectedType} = this.props
+  getSortOptions(selectedType) {
     const type = schema.get(selectedType)
-    const url = this.context.__internalRouter.resolveIntentLink('create', {
-      type: type.name
-    })
-    this.context.__internalRouter.navigateUrl(url)
+    return (type.sorting
+      ? DEFAULT_SORT_OPTIONS.concat(type.sorting)
+      : DEFAULT_SORT_OPTIONS).map(option => {
+        return {
+          ...option,
+          icon: SortIcon,
```

Comment:
Perhaps we should allow `icon` to be specified in sort options? It would make sense to allow say a clock/calendar icon for date sorting and a `Aa↓` icon for sorting by title, for instance.

Reply:
Totally! Will fix.